features/step_definitions/2.majas_darbs_steps.rb

```diff
@@ -0,0 +1,150 @@
+require 'rest-client'
+require 'test-unit'
+
+When(/^I create a new project$/) do
+    login_positive #first login to generate a cookie
+    @test_user.session_cookie.to_json
+
+    project_name = (0...10).map { (65 + rand(26)).chr }.join #generate random project name
+
+    project_payload = {
+        :name => project_name,
+        :type => "basic"}.to_json
+    response = RestClient::Request.execute(method: :post,
```

Comment:
I guess you could create the api helper file that we used for post, get and other methods